build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+plugins {
+    java
+    `maven-publish`
+    alias(libs.plugins.lavalink)
+}
+
+group = "org.example"
+version = "0.1.0"
+
+lavalinkPlugin {
+    name = "plugin-template"
+    apiVersion = libs.versions.lavalink.api
+    serverVersion = gitHash(libs.versions.lavalink.server)
+}
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(17)
+    }
+}
+
+tasks {
+    compileJava {
+        options.encoding = "UTF-8"
+    }
+}
+
+dependencies {
+    // add your dependencies here
```

Comment:
An example here would be good, particularly as it's not clear which dependency configuration to use

Reply:
You use the one you would use per gradle convention `implementation` or `compileOnly`

Reply:
So not really sure for what the example should be

Reply:
I suppose